src/Frontend/src/components/messages2/SagaDiagram/MessageDataBox.vue

```diff
@@ -28,16 +28,18 @@ const body = computed(() => props.messageData.body.data.value || "");
     <LoadingSpinner />
   </div>
   <div v-else-if="messageData.body.failed_to_load" class="message-data-box message-data-box-error">
-    <span class="message-data-box-text--error">An error occurred while retrieving the message data</span>
+    <span class="message-data-box-text--error" v-tippy="`Failed to load message data - there might be a connection issue or the message may no longer be available`">An error occurred while retrieving the message data</span>
   </div>
   <div v-else-if="!messageDataLoading && (!messageData.body.data.value || messageData.body.not_found)" class="message-data-box">
-    <span class="message-data-box-text--empty">Empty</span>
+    <span class="message-data-box-text--empty" v-tippy="`No message body data available`">Empty</span>
   </div>
   <div v-else-if="contentType.isSupported" class="message-data-box message-data-box-content">
     <MaximizableCodeEditor :model-value="body" :language="contentType.language" :readOnly="true" :showGutter="false" :modalTitle="modalTitle" />
   </div>
   <div v-else class="message-data-box message-data-box-error">
-    <span class="message-data-box-text--unsupported">Message body cannot be displayed because content type "{{ messageData.body.data.content_type }}" is not supported.</span>
+    <span class="message-data-box-text--unsupported" v-tippy="`Content type '${messageData.body.data.content_type}' cannot be displayed in the message viewer`"
```

Comment:
No need for a tootip here, we already explain why
```suggestion
    <span class="message-data-box-text--unsupported"
```